### PR TITLE
Change validation unique title

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
bugfix https://github.com/ProcessMaker/bpm/issues/512

- add unique validation title if it exists in the process in models Form, input output document, scripts.

### CODE REVIEW

File: ProcessMaker/Managers/FormsManager.php [hunk 1]
```diff
@@ -144,6 +146,18 @@ private function validate($data)
          */
 
         if (!isset($data['copy_import'])) {
+            $validator = Validator::make(
+                $data,
+                [
+                    //Validate title unique by process
+                    'title' => ['required', Rule::unique('forms')->where(function ($query) use ($data) {
```
Comment (velkymx) [inline ProcessMaker/Managers/FormsManager.php]: Please move the validation rules into the model.

File: ProcessMaker/Managers/InputDocumentManager.php [hunk 2]
```diff
@@ -107,6 +108,9 @@ private function validate($data, $update=false)
         $validator = Validator::make(
             $data,
             [
+                'title' => ['required', Rule::unique('input_documents')->where(function ($query) use ($data){
```
Comment (velkymx) [inline ProcessMaker/Managers/InputDocumentManager.php]: Move rules into the model

File: ProcessMaker/Managers/ScriptManager.php [hunk 3]
```diff
@@ -85,4 +90,30 @@ public function remove(Script $script): ?bool
...
+     */
+    private function validate($data)
+    {
+        /**
+         * @var $validator \Illuminate\Validation\Validator
+         */
+        $validator = Validator::make(
+            $data,
+            [
+                'title' => ['required', Rule::unique('scripts')->where(function ($query) use ($data){
```
Comment (velkymx) [inline ProcessMaker/Managers/ScriptManager.php]: move to model

**Comment by marcoAntonioNina** [comment]
The unique title validation was changed to models

**Pull request merged**